src/main/webapp/app/shared/layouts/main/main.component.ts

```diff
@@ -49,4 +65,10 @@ export class JhiMainComponent implements OnInit {
 
         this.themeService.initialize();
     }
+
+    private shouldShowSkeleton(url: string): boolean {
```

Comment:
I would also appreciate some code documentation here

Reply:
I added clarifying comments to both the property and `shouldShowSkeleton`.